source/Octopus.Cli/Util/CommandOutputProvider.cs

```diff
@@ -1,125 +1,41 @@
 ﻿using System;
-using System.IO;
-using System.Linq;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Converters;
 using Octopus.Cli.Diagnostics;
-using Octopus.Cli.Infrastructure;
 using Octopus.Client.Model;
 using Octopus.Client.Serialization;
+using Octopus.CommandLine;
 using Serilog;
-using Serilog.Events;
 
 namespace Octopus.Cli.Util
 {
-    public class CommandOutputProvider : ICommandOutputProvider
+    public interface IOctopusCliCommandOutputProvider : ICommandOutputProvider
```

Comment:
This is to split the old stuff that is OctopusCLI specific away from the generic `ICommandOutputProvider` interface that's now in the `Octopus.CommandLine` library.